api/v1alpha1/backstage_types.go

```diff
@@ -180,7 +180,7 @@ type Env struct {
 type RuntimeConfig struct {
 	// Name of ConfigMap containing Backstage runtime objects configuration
 	BackstageConfigName string `json:"backstageConfig,omitempty"`
-	// Name of ConfigMap containing LocalDb (P|ostgreSQL) runtime objects configuration
+	// Name of ConfigMap containing LocalDb (PostgreSQL) runtime objects configuration
```

Comment:
This would mean regenerating the bundle manifests.. Since this is not strictly related to RHIDP-1701, can this be done in a separate PR instead?